service/storage_service.cc

```diff
@@ -5000,13 +5002,27 @@ future<raft_topology_cmd_result> storage_service::raft_topology_cmd_handler(shar
                     //co_await sleep_abortable(_db.local().get_config().read_request_timeout_in_ms() * std::chrono::milliseconds(1), _abort_source);
                     result.status = raft_topology_cmd_result::command_status::success;
                 break;
+                case raft_topology_cmd::command::fence: {
+                    const auto version = get<topology::version_t>(cmd.param);
```

Comment:
Here I also think that the version parameter can be implicit and taken from the local topology object.